pkg/shp/cmd/build/create.go

```diff
@@ -3,9 +3,10 @@ package build
 import (
 	"fmt"
 
-	buildv1alpha1 "github.com/shipwright-io/build/pkg/apis/build/v1alpha1"
+	buildv1beta1 "github.com/shipwright-io/build/pkg/apis/build/v1beta1"
 	"github.com/spf13/cobra"
 
+	v1 "k8s.io/api/core/v1"
```

Comment:
For consistency, we should use "full name".
```suggestion
	corev1 "k8s.io/api/core/v1"
```